packages/stores/src/app.ts

```diff
@@ -0,0 +1,52 @@
+import { useLocalStorage } from '@vueuse/core';
+import type {} from '@vueuse/shared';
```

Comment:
Was this added on purpose?

Reply:
Yup! If you don't, you get a TypeScript error about how types aren't portable because of a dependency on a non-imported file (it's a bug in TS when using symlinked packages, eg PNPM)

Reply:
Hah, I see 😆